information-mediator/docs/main.md

```diff
@@ -0,0 +1,473 @@
+# Sandbox-Information-Mediator
+
+The Sandbox Information Mediator(IM) BB is implemented by [X-Road](https://github.com/nordic-institute/X-Road)
+
+## Introduction
+
+The Sandbox Information Mediator(IM) BB in implemented by running [X-Road Central Server](https://github.com/nordic-institute/X-Road/blob/develop/doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md) and [X-Road Security Server](https://github.com/nordic-institute/X-Road/blob/develop/doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md) in Kubernetes Cluster in the cloud. The initial goal is to implement it in as much cloud agnostic way as possible.
+
+## Tooling
+
+The tools used for building and deploying the IM to Kubernetes Cluster in the cloud:
+
+* [Helm charts](https://helm.sh/docs/topics/charts/)
+* [Docker](https://www.docker.com/) for X-Road containerization
+* [CircleCI](https://circleci.com/) for CI/CD
+
+## Process
+
+1. Running X-Road Central Server in a Docker container
+
+   ``` bash 
+    $ docker run -d -p 4000:4000 -p 4001:80 -p 4002:9998 --name cs <XROAD_CENTRAL_SERVER_IMAGE> 
+    ```
+
+    * XROAD_CENTRAL_SERVER_IMAGE - [official NIIS X-Road Central Server Dockerhub image](https://hub.docker.com/r/niis/xroad-central-server)
+
+2. Running X-Road Security Servers in Docker containers
+
+   ``` bash
+    $ docker run -d -p 4100:4000 -p 8081:8080 --name ssm -e XROAD_TOKEN_PIN=<XROAD_TOKEN_PIN> <XROAD_SECURITY_SERVER_IMAGE>
+    $ docker run -d -p 4200:4000 -p 8082:8080 --name ssc -e XROAD_TOKEN_PIN=<XROAD_TOKEN_PIN> <XROAD_SECURITY_SERVER_IMAGE>
+    $ docker run -d -p 4300:4000 -p 8083:8080 --name ssp -e XROAD_TOKEN_PIN=<XROAD_TOKEN_PIN> <XROAD_SECURITY_SERVER_IMAGE> 
+   ```
+
+    * XROAD_TOKEN_PIN - X-Road pin token, e.g. 1234
+    * XROAD_SECURITY_SERVER_IMAGE - [official NIIS X-Road Security Server Dockerhub image](https://hub.docker.com/r/niis/xroad-security-server)
+
+3. Substituting Docker entrypoint.sh script in docker containers with a custom entrypoint.sh script
+
+   for the Central Server container:
+   ``` bash
+    $ docker cp information-mediator/scripts/central-server-entrypoint.sh <CID>:/root/entrypoint.sh
+   ```
+
+   and for the Security Server container (the same has to be done for each Security Server container if there are multiple):
+   ``` bash
+    $ docker cp information-mediator/scripts/security-server-entrypoint.sh <CID>/root/entrypoint.sh 
+   ```
+
+    * CID - docker container id (can be obtained by for example by ``` $ export CID=$(docker ps -aqf "name=cs") ``` where the  ```name``` is the container name
+
+   **NOTE**
+   The [central-server-entrypoint.sh](information-mediator/scripts/central-server-entrypoint.sh) and [security-server-entrypoint.sh](information-mediator/scripts/security-server-entrypoint.sh) scripts contain customizations to the original Docker entrypoint.sh script to configure the X-Road components to use remote database
```

Comment:
These links do not work anymore as you have moved the documentation.

Reply:
Fix c84a4f112a5053772e60042ca260d90705e09962